scripts/service_migration.py

```diff
@@ -0,0 +1,89 @@
+import copy
+from kubernetes import client, config
+from kubernetes.client.exceptions import ApiException
+import argparse
+import os.path
+from pprint import pprint
+
+IP_ANNOTATION = 'load-balancer-ips'
+def update_annot(v1, svc):    
+    '''
+    ************************************************************************
+    Take dictionary of original service, copies Service.status.load_balancer.ingress[0].ip to Service.metadata.annotations and patches Service
+    --------------
+    Arguments:
+    v1 :: CoreV1Api object
+    svc :: original service as dictionary
+    --------------
+    Returns:
+    None
+    ************************************************************************
+    '''
+    
+    svc['metadata']['annotations'][IP_ANNOTATION] = svc['status']['load_balancer']['ingress'][0]['ip']
```

Comment:
Better to check the presence of keys `status`, `load_balancer`, `ip` and length of svc.

Reply:
Added check for existence of each key, and whether said keys hold null/empty values